### PR TITLE
nerfs Fungal TB

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
fix: people be bitching about fungal TB killing way too fuckin fast so I like nerfed the shit outta it, it should be survivable now, also the cure's easier lol
/:cl:

### CODE REVIEW

File: code/datums/diseases/tuberculosis.dm [hunk 1]
```diff
@@ -3,11 +3,11 @@
 	name = "Fungal tuberculosis"
 	max_stages = 5
 	spread_text = "Airborne"
-	cure_text = "Spaceacillin & salbutamol"
-	cures = list("spaceacillin", "salbutamol")
+	cure_text = "Pan-Galactic Gargle Blaster"
+	cures = list("gargleblaster")
```
Comment (praisenarsie) [inline code/datums/diseases/tuberculosis.dm]: you made the cure a bit harder but ok

Comment (Iamgoofball) [reply code/datums/diseases/tuberculosis.dm]: bartending is easy

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

File: code/datums/diseases/tuberculosis.dm (hunk 1 above)
Comment (81Denton) [reply code/datums/diseases/tuberculosis.dm]: What are you smoking, gargleblaster is probably the most common drink and easy as hell to mix
Even the dispensers are easy to reach

**Comment by Buggy123** [comment]
We need to check goofball into a psych ward he's gone mad

**Comment by optimumtact** [comment]
Your PR has been closed due to having a misleading or incorrect title, adjust the title and ping me to reopen it or make a new PR with a properly descriptive title and body content.

**Comment by deranging** [comment]
oranges ur gay lol

**Comment by Iamgoofball** [comment]
oranges you fuck I can't edit titles on mobile stop being dumb

**Comment by optimumtact** [comment]
then ping when it's fixed

**Pull request closed without merging**